openmetadata-docs/content/v1.4.x/deployment/security/saml/docker.md

```diff
@@ -0,0 +1,52 @@
+---
+title: SAML SSO for Docker
+slug: /deployment/security/saml/docker
+---
+
+# Keycloak SSO for Docker
+
+To enable security for the Docker deployment, follow the next steps:
+
+## 1. Create an .env file
+
+Create an `openmetadata_saml.env` file and add the following contents as an example. Use the information
+generated when setting up the account.
+
+The configuration below already uses the presets shown in the example of SAML configurations, you can change to yours.
+
+```shell
+# OpenMetadata Server IDP Configuration
+SAML_IDP_ENTITY_ID=https://mocksaml.com/api/saml/sso
+SAML_IDP_SSO_LOGIN_URL=https://saml.example.com/entityid
+SAML_IDP_CERTIFICATE=/path/to/the/certificate
+SAML_AUTHORITY_URL=http://localhost:8585/api/v1/saml/login
+SAML_IDP_NAME_ID=urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress
```

Comment:
Let's add a note that if we are passing the env from external file, and if the env is commented in the docker-compose, the env won't be taken into consideration, the commented part in the docker-compose should be reverted back / uncommented to work